packages/blade/package.json

```diff
@@ -58,6 +58,7 @@
   },
   "dependencies": {
     "@babel/runtime": "7.16.5",
+    "outdent": "0.8.0",
```

Comment:
Do we really need this? Do you forsee us using this elsewhere too?

Reply:
We can use this on all the Highlight's in storybook. 

Writing in a single line is a bit messy. This override example is the biggest tho 

```tsx
      <Highlight language="tsx">
        {outdent`
          import { Badge } from '@razorpay/blade/components';
          import type { BadgeProps } from '@razorpay/blade/components';
        `}
      </Highlight>
```

Reply:
Okay. Can we have it as devDependencies then?

Reply:
I think that would break on storybook deploy right?